src/subdomains/supporting/payment/dto/transaction.dto.ts

```diff
@@ -46,6 +46,7 @@ export enum TransactionReason {
   CHF_ABROAD_NOT_ALLOWED = 'ChfAbroadNotAllowed',
   ASSET_KYC_NEEDED = 'AssetKycNeeded',
   CARD_NAME_MISMATCH = 'CardNameMismatch',
+  USER_DEACTIVATED = 'UserDeactivated',
 }
```

Comment:
Maybe we should call it USER_DELETED, because we externally speak of deletion, not deactivation?

Reply:
Yeah would be make sense, I clarify that